speedrun-3.html

```diff
@@ -70,6 +70,7 @@
 </div>
 <!-- margin-top 0 em because contentContainer already has a margin-top.-->
 <article id="guide" class="main" style="margin-top:0em">
+<div class="warning">This was a developer test page that converted the V3 guide content into an interactive version. This version is completely untested and is not intended to be used in an actual speedrun.</div>
```

Comment:
this page is not completely untested. I've done around 75% of the run with it

Reply:
I updated the wording to be "not completely tested". Eventually it would be nice to make full versions of each previous guide with these warnings just stating that the version is out of date and may have logic errors but that's a v5 thing.